xAPI.md

```diff
@@ -509,12 +509,56 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 8.0 Getting More Out of xAPI
```

Comment:
How about "Using xAPI within a Community of Practice" and then make 'Communities of Practice' the first section?